manifests/charts/cloudcore/values.yaml

```diff
@@ -3,10 +3,14 @@
 # Declare variables to be passed into your templates.
 appVersion: "1.12.0"
 
+global:
+  imageRegistry: ""
+
 cloudCore:
   replicaCount: 1
   hostNetWork: "true"
   image:
+    registry: docker.io
```

Comment:
if we specify this two registry both, what's the expected behavior?

Reply:
If the images come from different registry (which is rare), we do not need to configure global.
 If they come from the same registry, you can configure global. 
Global first

Reply:
I agree with @gy95, we had better simplify the codes to avoid bringing confusion.

Reply:
ok, I will close it